src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1484,6 +1484,87 @@ def db_threat_detection_policy_update(
     return instance
 
 
+def db_recommended_sensitivity_labels_list(
+        client,
+        database_name,
+        server_name,
+        resource_group_name):
+    return client.list_recommended_by_database(resource_group_name, server_name, database_name)
+
+
+def db_sensitivity_label_update(
+        cmd,
+        client,
+        database_name,
+        server_name,
+        schema_name,
+        table_name,
+        column_name,
+        resource_group_name,
+        label_name=None,
+        information_type=None):
+    '''
+    Updates a sensitivity label. Custom update function to apply parameters to instance.
+    '''
+
+    # Get the information protection policy
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azure.mgmt.security import SecurityCenter
+    from msrestazure.azure_exceptions import CloudError
+
+    securityCenterclient = get_mgmt_service_client(cmd.cli_ctx, SecurityCenter, asc_location="centralus")
+
+    informationProtectionPolicy = securityCenterclient.information_protection_policies.get(
+        scope='/providers/Microsoft.Management/managementGroups/{}'.format(_get_tenant_id()),
+        information_protection_policy_name="effective")
+
+    params = {
```

Comment:
Instead of building a dictionary, you should be able to use the sensitivity label class from python sdk